.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "workbench.colorCustomizations": {}
+}
```

Comment:
Looks like VS Code files are sneaking in. Adding a `.vscode` entry to the .gitignore file should do the trick.